internal/cli/users.go

```diff
@@ -516,7 +522,7 @@ func listUserBlocksCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List brute-force protection blocks for a given user",
 		Long:    "List brute-force protection blocks for a given user.",
-		Example: "auth0 users blocks list <user-id>",
+		Example: `  auth0 users blocks list <user-id>`,
```

Comment:
Consider adding in a future PR an example with the `ls` alias, like in other `list` commands.

Reply:
Consider as well adding an example with the `--json` flag, like in other similar commands.